docs/experiments.rst

```diff
@@ -0,0 +1,318 @@
+.. _Experiments:
+
+Experiments
+===========
+
+You might want to take a look at the following pages before exploring experiments:
+
+- :ref:`QuickStart`
+- :ref:`AtVariables`
+- :ref:`Instances`
+- :ref:`Builds`
+- :ref:`Revisions`
+- :ref:`Variants`
+
+On this page we describe how to specify experiments in our ``experiments.yml`` file. We will see how
+to use builds in experiments and enable options such as repeating experiments or setting a timer.
```

Comment:
`[...] setting a timeout.`

Reply:
Done. I also changed it in the "Timeout" section, as I used "timer" instead of "timeout" in some parts.